test/verify/check-packagekit

```diff
@@ -54,9 +54,15 @@ class TestUpdates(MachineCase):
         m.start_cockpit()
         b.login_and_go("/updates")
 
+        # PK starts from a blank state, thus should force refresh
+        b.wait_in_text("#state", "Refreshing")
+        b.wait_present("#app div.progress-bar")
+
         # no repositories at all, thus no updates
         b.wait_present(".content-header-extra td button")
         b.wait_in_text("#state", "No updates pending")
+        b.wait_present(".content-header-extra td.text-right span")
+        self.assertEqual(b.text(".content-header-extra td.text-right span"), "Last checked: a few seconds ago")
```

Comment:
Actually no, this is safe. We set the time since refresh to exactly "0" after calling `RefreshCache()`, so this isn't subject to tests taking a longer time. We just can't assert it in tests where we call `pkcon refresh` manually before loading the page.